hack/create-user-workspace.sh

```diff
@@ -39,5 +39,18 @@ kubectl ws create ${USER_APPSTUDIO_WORKSPACE}  --ignore-existing --type root:uni
 kubectl kustomize ${ROOT}/apibindings/${SERVICE_NAME}/ | sed "s|\${APPSTUDIO_SP_WORKSPACE}|${APPSTUDIO_SP_WORKSPACE}|g;s|\${HACBS_SP_WORKSPACE}|${HACBS_SP_WORKSPACE}|g" | \
   kubectl apply -f -
 
+echo
+echo "Patching APIBindings to accept all permission claims:"
+API_BINDINGS=$(kubectl get apibindings.apis.kcp.dev -l provided-by=infra-deployments -o name)
+for API_BINDING in ${API_BINDINGS}
+do
+  EXPORT_PERMISSION_CLAIMS=$(kubectl get ${API_BINDING} -o jsonpath='{.status.exportPermissionClaims}')
+  for EXPORT_CLAIM in $(echo ${EXPORT_PERMISSION_CLAIMS} | jq -c '.[]')
+  do
+    ACCEPTED_CLAIMS=${ACCEPTED_CLAIMS}$(echo ${EXPORT_CLAIM} | jq '. += {"state": "Accepted"}' | jq -c)","
```

Comment:
ACCEPTED_CLAIMS is global variable so it's used by all API_BINIDINGS in the cycle, it should be emptied at the beginning of first for cycle